### ISSUE TITLE
Something causes segfault

### ISSUE THREAD

**Issue by Aciz** [etjump/etjump#547]
Happens somewhere between `G_LogPrintf` and the end of the function. [g_client.cpp, L2049-2084](https://github.com/etjump/etjump/blob/master/src/game/g_client.cpp#L2049)
```cpp
	G_LogPrintf("ClientBegin: %i\n", clientNum);

	// count current clients and rank for scoreboard
	CalculateRanks();

	// No surface determined yet.
	ent->surfaceFlags = 0;

	ETJump::saveSystem->loadPositionsFromDatabase(ent);
	OnClientBegin(ent);
	if (level.hasTimerun)
	{
		trap_SendServerCommand(clientNum, "hasTimerun 1");
	}
	else
	{
		trap_SendServerCommand(clientNum, "hasTimerun 0");
	}
	for (i = 0; i < MAX_TOKENS_PER_DIFFICULTY; ++i)
	{
		ent->client->pers.collectedEasyTokens[i]   = qfalse;
		ent->client->pers.collectedMediumTokens[i] = qfalse;
		ent->client->pers.collectedHardTokens[i]   = qfalse;
	}
	ent->client->pers.tokenCollectionStartTime = level.time;

	if (!ent->client->sess.receivedTimerunStates)
	{
		TimerunConnectNotify(ent);
		ent->client->sess.receivedTimerunStates = qtrue;
	}

	ent->client->pers.previousSetHealthTime = 0;

	
}
```

```
Aug 31 17:13:31 trickjump-net-v2 etded[17953]: ClientUserinfoChanged: 9 ^E^E^E^E^E^E^E^E
Aug 31 17:13:31 trickjump-net-v2 etded[17953]: ClientBegin: 9
Aug 31 17:13:34 trickjump-net-v2 etded[17953]: Received signal 11, exiting...
Aug 31 17:13:44 trickjump-net-v2 systemd[1]: et-27960.service: Service hold-off time over, scheduling restart.
Aug 31 17:13:44 trickjump-net-v2 systemd[1]: et-27960.service: Scheduled restart job, restart counter is at 7.
```

**Comment by Aciz** [comment]
Not ClientBegin.

**Comment by Aciz** [comment]
Seems to be resolved via #549
